functions.php

```diff
@@ -60,3 +60,148 @@ function showTimeGap($datePoint)
 
     return $num . ' ' . $word . ' назад';
 }
+
+/**
+ * Возвращает ресура соединения с указанной базой данных.
+ * @param string $db_host Строка, имя сервера, по умолчанию localhost
+ * @param string $db_user Строка, имя для входа в базу данных
+ * @param string $db_password Строка, пароль для входа в базу
+ * @param string $db_name Строка, имя базы данных
+ *
+ * @return mysqli $con ресурс соединения при удачном соединении или строку с ошибкой
+ */
+function db_connect($db_host = 'localhost', $db_user, $db_password, $db_name)
```

Comment:
Тайпхинты для аргументов `string $db_host = 'localhost', string $db_user`, ...

Reply:
Тайпхинты указал 66-69 строки

```suggestion
* @param string $db_host имя сервера, по умолчанию localhost
 * @param string $db_user имя для входа в базу данных
 * @param string $db_password пароль для входа в базу
 * @param string $db_name имя базы данных
```